modules/bluebillywigBidAdapter.js

```diff
@@ -52,10 +51,10 @@ const BB_HELPERS = {
     else if (Array.isArray(adServerCur) && adServerCur.length) request.cur = [adServerCur[0]];
   },
   addUserIds: function(request, validBidRequests) {
-    const bidUserId = deepAccess(validBidRequests, '0.userId');
-    const eids = createEidsArray(bidUserId);
+    const bidUserId = deepAccess(validBidRequests, '0.userIdAsEids');
+    const eids = bidUserId;
```

Comment:
this line is doing nothing except renaming; you can move the name up (`const eids = deepAccess(...`)